hackathon/spacecraft/app.config.js

```diff
@@ -40,4 +40,24 @@ module.exports = {
       projectId: "012accc3-4ce5-4bae-9f4d-2f842489f07a",
     },
   },
+  plugins: [
+    [
+      "app-icon-badge",
+      {
+        enabled: false, // enable/ disable the plugin based on the environment (usually disabled for production builds)
```

Comment:
I think we can automate this like
```
enabled: process.env.ENVIRONMENT === 'production' ? false : true
```
Trigger an EAS build and test it to be sure it works